devtools/create_lure/create_lure_dat.cpp

```diff
@@ -1400,6 +1427,17 @@ const char *spanishTextStrings[NUM_TEXT_ENTRIES] = {
 	"el ", "la ", "los ", "las ", "este ", "esta ", "estos ", "estas ", nullptr, nullptr
 };
 
+const char *russianTextStrings[NUM_TEXT_ENTRIES] = {
+	"dpznm", nullptr, "yf;fnm", "gjnzyenm", "bcgjkmpjdfnm", "jnrhsnm", "pfrhsnm", "pfgthtnm", "jngthtnm", "ghbvtybnm",
```

Comment:
Would not hurt to add the actual Russian as comments, but probably it will not be trivial to decipher :D